routes/index.js

```diff
@@ -563,7 +572,7 @@ router.put('/discover', function(req, res, next) {
     //apply 'show songs in' filter
     var loadMore = function(songs2d, done) {
         if (langShown !== 'all') {
-            songs2d = songs2d.filter((s) => s.lang === langShown)
+            songs2d = songs2d.filter((s) => s.lang == langShown)
```

Comment:
Also this is a big file at 550 lines long. Can a bunch of these methods under this route be refactored into a separate object/file?